<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Guess My Number : a Qt Widgets app</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>42</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuess">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Guess</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartOver">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start over</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
